packages/expo-tracking-transparency/ios/TrackingTransparencyPermissionRequester.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import AppTrackingTransparency
+import ExpoModulesCore
+
+public class TrackingTransparencyPermissionRequester: NSObject, EXPermissionsRequester {
+  static public func permissionType() -> String {
+    return "appTracking"
+  }
+
+  public func requestPermissions(resolver resolve: @escaping EXPromiseResolveBlock, rejecter reject: EXPromiseRejectBlock) {
+    if #available(iOS 14, *) {
+      ATTrackingManager.requestTrackingAuthorization() { [weak self] _ in
+        resolve(self?.getPermissions());
+      }
+    } else {
+      resolve(self.getPermissions());
+    }
+  }
+
+  public func getPermissions() -> [AnyHashable: Any] {
+    var status: EXPermissionStatus
+    
+    if #available(iOS 14, *) {
+      var systemStatus = ATTrackingManager.trackingAuthorizationStatus
+      switch systemStatus {
+      case .authorized:
+        status = EXPermissionStatusGranted
+      case .restricted,
+           .denied:
+        status = EXPermissionStatusDenied
+      case .notDetermined:
+        fallthrough
+      @unknown default:
+        status = EXPermissionStatusUndetermined
+      }
+    } else {
+      status = EXPermissionStatusGranted
+    }
+
```

Comment:
Should I put here something like in [`ImagePickerPermissionRequesters.swift`](https://github.com/expo/expo/blob/main/packages/expo-image-picker/ios/ImagePickerPermissionRequesters.swift):

https://github.com/expo/expo/blob/017a717d6e53504db019327a27ceb1047f8f8805/packages/expo-image-picker/ios/ImagePickerPermissionRequesters.swift#L20-L28

Reply:
Yeah, I think we should. There is a `NSUserTrackingUsageDescription` that is probably required